test/files/skelebot.yaml

```diff
@@ -11,6 +11,7 @@ components:
     url: https://repository.cars.com/artifactory
   jupyter:
     folder: notebooks/
+    mappings: .
```

Comment:
This should accept a list the same way we do for jobs:
```
mappings:
- .
- ~/test:/app/test
```

Reply:
I updated to
```
mappings:
- ~/test:/app/test
```
because `.`  is always added to mapping and no need to specify it in the config. let me know if it works.